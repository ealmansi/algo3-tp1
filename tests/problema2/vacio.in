# n
# d1 t1
# d2 t2
# ...
# dn tn
# ... mÃ¡s instancias
# 0

0