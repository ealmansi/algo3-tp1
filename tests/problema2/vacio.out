# i1 ... in P
# ... mÃ¡s instancias
